leshan-server-redis/src/main/java/org/eclipse/leshan/server/redis/RedisSecurityStore.java

```diff
@@ -49,7 +49,13 @@ public class RedisSecurityStore implements EditableSecurityStore {
 
     private final List<SecurityStoreListener> listeners = new CopyOnWriteArrayList<>();
 
-    private RedisSecurityStore(Builder builder) {
+    public RedisSecurityStore(Pool<Jedis> pool) {
+        this.pool = pool;
+        this.securityInfoByEndpointPrefix = "SECSTORE#SEC#EP#";
+        this.endpointByPskIdKey = "SECSTORE#EP#PSKID";
```

Comment:
maybe you can use 
```java
this(new Builder(pool).build());
```

So there is only 1 place where we define default value.
I know we will create a builder instance  just to do that but this is something which is not called often

Reply:
Certainly makes sense. I added a `protected` constructor to be able to use `this(...)` here.

Reply:
Uups my mistake I wanted to say : 
``` java
this(new Builder(pool)); 
```

So no need new constructor;

Reply:
Good, it's exactly what I wanted to do at first.